app/views/ConfirmSubmission.scala.html

```diff
@@ -112,25 +112,32 @@ <h2 class="govuk-heading-m">@messages("confirm_submission.declarationHeading")</
   if(nIProtocolEnabled) ".NIProtocol" else ""
 }
 
-@mainTemplate(pageTitle = messages("confirm_submission.title"), appConfig = appConfig, user = Some(user)) {
+@title = @{
+  messages("confirm_submission.heading") +
```

Comment:
This text is different in the screenshot - "Confirm return details"
(do we have Welsh for this? may have to request it if we've overlooked this)